.circleci/config.yml

```diff
@@ -615,32 +652,70 @@ jobs:
       - run_e2e_test:
           env_name: local
 
+  cdr-make-bq-denormalized-tables:
+    <<: *java_defaults
+    steps:
+      - checkout_init_git
+      - manage_api_cache:
+          restore: true
+      - activate_service_account_credential
+      - make_bq_denormalized_tables:
+          data_browser: false
+
+  cdr-make-bq-denormalized-tables-for-data-browser:
+    <<: *java_defaults
+    steps:
+      - checkout_init_git
+      - manage_api_cache:
+          restore: true
+      - activate_service_account_credential
+      - make_bq_denormalized_tables:
+          data_browser: true
+
+  cdr-generate-private-counts:
+    <<: *java_defaults
+    steps:
+      - checkout_init_git
+      - manage_api_cache:
+          restore: true
+      - activate_service_account_credential
+      - run:
+          name: Make BQ denormalized tables
+          working_directory: ~/workbench/api
+          # TODO: un-echo. Will need to discriminate which SA to use.
+          command: echo "./project.rb generate-private-cdr-counts " \
+            "--bq-project        << pipeline.parameters.cdr_source_project >> " \
+            "--bq-dataset        << pipeline.parameters.cdr_source_dataset >> " \
+            "--workbench-project << pipeline.parameters.cdr_dest_project >> " \
+            "--cdr-version       << pipeline.parameters.cdr_version_db_name >> " \
+            "--bucket            << pipeline.parameters.cdr_sql_bucket >> "
+
 workflows:
-  build-test-deploy:
-    jobs:
-      # Always run basic test/lint/compilation (open PRs, master merge).
-      # Note: by default tags are not picked up.
-      - api-local-test
-      - api-unit-test
-      - ui-unit-test
-      - api-bigquery-test
-      - api-integration-test
-      - puppeteer-e2e-local-ui
-      # Run deployment to "test" on master merges.
-      - api-deploy-to-test:
-          filters: *filter_only_master_branch
-          requires:
-            - api-unit-test
-      - ui-deploy-to-test:
-          filters: *filter_only_master_branch
-          requires:
-            - ui-unit-test
-      # On master branch merges, run Puppeteer tests after ui and api deployed to "test" env successfully.
-      - puppeteer-e2e-test:
-          filters: *filter_only_master_branch
-          requires:
-            - api-deploy-to-test
-            - ui-deploy-to-test
+  # build-test-deploy:
```

Comment:
This avoids CircleCI presubmit triggering on my frequent pushes to this PR. Alternatively [skip ci] in git commit message